test/unittests/tools/qbs/test_qbsdeps.py

```diff
@@ -0,0 +1,100 @@
+import mock
```

Comment:
I have been checking the code above and it is not clear how it will be consumed by the qbs build system.

I think that at least 1 full ``functional`` test that does a real build with dependencies (the dependency itself can be the one ``matrix`` from the fixture already existing) would be very recommended.

Reply:
> it is not clear how it will be consumed by the qbs build system.

We will pass the path to the conan build dir to qbs via CLI. This requires additional changes to Qbs in order to be done automatically, but I want to merge this first - I prefer to keep changes isolated. At least, users can pass required args manually, see examples here:
https://codereview.qt-project.org/c/qbs/qbs/+/336750/44/doc/reference/module-providers/conan-module-provider.qdoc

Reply:
Moved to integration tests, indeed they are much simpler.